Shared/sdk/net/rpc_enums.h

```diff
@@ -104,7 +104,7 @@ enum eElementRPCFunctions
     TAKE_ALL_WEAPONS,
     SET_WEAPON_AMMO,
     SET_WEAPON_SLOT,
-
+    
```

Comment:
```suggestion

```

Reply:
Sorry, but I don't understand why remove the indent. That was before my pull request. And sorry for my poor English

Reply:
The indent shows up here https://github.com/multitheftauto/mtasa-blue/pull/1143/files, which means it was introduced by your pull request